zhaquirks/philips/hue_light.py

```diff
@@ -169,3 +169,36 @@
     .replaces(PhilipsHueLightCluster, endpoint_id=11)
     .add_to_registry()
 )
+
+(
+    QuirkBuilder()
+    .applies_to(SIGNIFY, "LCD003")
+    .friendly_name(
+        model="Hue Akari Downlight",
```

Comment:
HA generally wants only the first letter of the first word to be capitalized, excluding proper names and abbreviations. The other names above might not be ideal either, but they should give you an idea. (Maybe ignore the capitalized "Pedestal". That doesn't make a lot of sense to me right now.)

For this, what about?
```suggestion
        model="Hue Akari downlight",
```